### DISCUSSION TITLE
HaxeFlixel Source Code: Where are `FlxBasic` even proccessed?

### DISCUSSION THREAD

**Discussion by sol-vin** [HaxeFlixel/flixel#3187] [Q&A] [answered]
Looking through the source code for HaxeFlixel and was wondering where objects in the FlxState get processed. I've checked through `FlxState` and `FlxGame` and I never found a place where `FlxBasic` are gone over and processed.

**Comment by Geokureli** [comment, marked as answer]
explain what you mean when you say "processed"

if you mean "where does it call update and draw?" FlxState extends FlxGroup, which updates it's members, here:
https://github.com/HaxeFlixel/flixel/blob/dev/flixel/group/FlxGroup.hx#L155-L184

**Comment by sol-vin** [comment]
Sorry I could have been clearer. Basically where a `FlxBasic`'s `update` and `draw` methods are called. Or at the very least where the collection of `FlxBasic` is gone over when running an update on an `FlxState`

**Reply by Geokureli** [reply]
I edited my previous comment with this info already
